source/extensions/filters/http/cache/cache_filter.h

```diff
@@ -126,10 +131,12 @@ class CacheFilter : public Http::PassThroughFilter,
   // being cancelled.
   InsertStatus insertStatus() const;
 
+  std::unique_ptr<CacheInsertQueue> insert_queue_;
```

Comment:
think it's worth a comment on ownership/lifetime here since it's not really as simple as unique pointer?